### ISSUE TITLE
Artifact of Chaos

### ISSUE THREAD

**Issue by peterbozso** [peterbozso/ror2arena#4]
Disable the Artifact of Chaos immediately after the Champion is announced. I've come across many times where I would die to my own items/abilities after the Arena fight.

Also consider implementing PvP without using Artifact of Chaos. This way, players couldn't die from their own damage. For example, explosions for melee survivors can be a big handicap.

**Comment by peterbozso** [comment]
@AJ9844 disabling the Artifact of Chaos immediately after the Champion is announced is included in 0.2.0 now.

**Comment by piebaker** [comment]
Seems melee characters seem to have a hard time with FF, namely tested with the miner mod and mercenary

**Comment by peterbozso** [comment]
@piebaker Yes, it's high on my priority list to find a solution for this. (The third task in this issue is all about that.)
